ui/src/hooks/useKeyPress.ts

```diff
@@ -10,8 +10,12 @@ const useKeyPress = (eventConfigs, callback, node = null) => {
   // handle what happens on key press
   const handleKeyPress = useCallback(
     (event) => {
+      event.preventDefault();
```

Comment:
why do you need this twice?
and are you sure you need this at all? I think maybe stopPropagation should be enough in this case